lib/tasks/evm_settings.rake

```diff
@@ -81,17 +81,28 @@ module EvmSettings
   private_class_method :keyval_to_hash
 
   def self.str_to_value(value)
-    return true  if value =~ /true/i
-    return false if value =~ /false/i
-    return nil   if value =~ /nil/
-    value
+    case
```

Comment:
This is a bug because it should be `case value`, but this was the thing you checked elsewhere, so I presume this goes away.

Reply:
yea. an improved version of this is alraedy merged into master. I will rebase it away * poof *